docs/n00b-gettingStarted.md

```diff
@@ -1,42 +1,47 @@
+Thankfully the mermaid renderer is very accessible, in essence it is a piece of javascript that can be called. 
 
-The requirement is on the part of the web browser. Modern web browsers, such as Firefox, Chrome and Safari, can render mermaid. But Internet Explorer cannot. The web browser also needs access to the online mermaid renderer which it downloads from https://cdn.jsdelivr.net/npm/mermaid
+Most widely used web browsers, such as Firefox, Chrome and Safari, can render mermaid, Internet Explorer however cannot. The web browser also needs access to the online mermaid renderer which it downloads from https://cdn.jsdelivr.net/npm/mermaid
 
-For an easy introduction, here follows three practical examples using:
-1. an online mermaid editor
-2. a mermaid plugin
-3. a generic web server of your choosing
+# For beginners, there are three relatively easy ways you can use mermaid:
+1. Using the mermaid [live editor](https://mermaid-js.github.io/mermaid-live-editor/)
+2. Using a mermaid plugin, such as that for Confluence or [Atom](https://atom.io/packages/atom-mermaid).
+3. Calling mermaid renderer with HTML, deployed in a friendly browser. 
 
-Following either of these examples, you can get started with converting your own mermaid code into web diagrams.
+# Following either of these examples, you can get started with creating your own diagrams using mermaid code.
 
-## the mermaid live editor
+## 1. The mermaid live editor:
 
-The quickest way to get started with mermaid is to visit [The mermaid live editor](https://mermaidjs.github.io/mermaid-live-editor).
+ A great way to get started with mermaid is to visit [The mermaid live editor](https://mermaidjs.github.io/mermaid-live-editor).
 
-In the `Code` section one can write or edit raw mermaid code, and instantly `Preview` the rendered result.
+In the `Code` section one can write or edit raw mermaid code, and instantly `Preview` the rendered result on the panel beside it.
 
-This is a great way to get started.
+![Flowchart](./img/n00b-liveEditor.png) 
 
-It is also the easiest way to develop diagrams, the code of which can be pasted straight into documentation.
 
-![Flowchart](./img/n00b-liveEditor.png)
+You can also copy the code from the code section and paste it into either a mermaid plugin or in inside an html file, which will be taught in numbers 2 and 3. 
+
+It is also an easier way to develop diagrams. You can also click "Copy Markdown" to copy the markdown code for the diagram, that can then be pasted directly into your documentation. 
+
+![Flowchart](./img/liveEditorOptions.png)
 
 The `Mermaid configuration` is for controlling mermaid behaviour. An easy introduction to mermaid configuration is found in the [Advanced usage](n00b-advanced.md) section. A complete configuration reference cataloguing default values is found on the [mermaidAPI](mermaidAPI.md) page.
 
 
-## mermaid using plugins
+## 2. Using mermaid plugins:
```

Comment:
Maybe refer for the full list to :  docs/integrations.md
https://github.com/mermaid-js/mermaid/pull/1278/files (for a bigger list)

Reply:
Excellent, just learning how to incorporate your suggestion...

Reply:
added, the link to integrations. thanks for the feedback.

Reply:
thank you for the feedback!